### ISSUE TITLE
moe loss bug

### ISSUE THREAD

**Issue by mst272** [modelscope/ms-swift#5375]
**Describe the bug**
I found that swift calculates the loss of moe:
```
            if self.model.model_info.is_moe_model and self.args.router_aux_loss_coef is not None:
                aux_loss = outputs.get('aux_loss')
                if aux_loss is not None:
                    loss = loss + self.args.router_aux_loss_coef * aux_loss.to(loss.device)
```
If router_aux_loss_coef is set, the final loss is the loss returned by the transformer trainer output + aux_loss.

However, in the implementation of qwen3 moe in the Transformer, it can be seen that the final loss has already added the aux_loss.
https://github.com/huggingface/transformers/blob/31ab7168ff7e07f61c90134e5238c4d97606aa70/src/transformers/models/qwen3_moe/modeling_qwen3_moe.py#L686

So is this a bug? Or is there a misunderstanding on my part somewhere?

**Your hardware and system info**
Write your system info like CUDA version/system/GPU/torch version here(在这里给出硬件信息和系统信息，如CUDA版本，系统，GPU型号和torch版本等)

**Additional context**
Add any other context about the problem here(在这里补充其他信息)

**Comment by Jintao-Huang** [comment]
When this branch is taken, the loss is not computed within Transformers; instead, it is calculated by an external plugin.

**Comment by mst272** [comment]
thanks
